### PR TITLE
Add logic for condition details menu

### LINKED ISSUE #205: [FE] Action buttons inside of the Condition detail page do not work

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #205

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr232--conditionaltokensexplorer.review.gnosisdev.com

**Review by gabitoesmiapodo — APPROVED** [review]
Seems to work fine.

I'm not sure if all of any of these actions require the user to be connected? For example, I don't think you should able to split a position unless you are connected. You get there and nothing tells you that you need to.

This may be part of a bigger issue, though. Take a look into that and please report back, we may need to address this in a separate issue (I think @mariano-aguero worked on this, but I'm not sure...) 

In short: Every section and action that requires for the user to be connected should check for it and (possibly) trigger the connect modal.

**Comment by hpmaxi** [comment]
@gabitoesmiapodo 
> In short: Every section and action that requires the user to be connected should check for it and (possibly) trigger the connect modal.

Yes I think is part of a bigger issue. You don't get the connect modal until you try to submit the form, except with the sections with an unlock button.  I would go for  the same behaviour without asking for connection if the page is loaded from the menu but first I'll check with @mariano-aguero if something like that was reported already.

**Comment by gabitoesmiapodo** [comment]
@hpmaxi In that case we should

a) Ask the user to connect before entering the section or...
b) ... ask the user to connect when clicking on the Submit / Split / Whatever button.

Please discuss it with Mariano and create an issue for that.

**Comment by elena-zh** [comment]
Looks good to me.

**Review by elena-zh — APPROVED** [review]
